src/ApiService/ApiService/onefuzzlib/notifications/Ado.cs

```diff
@@ -19,6 +19,7 @@ public class Ado : NotificationsBase, IAdo {
     // https://github.com/MicrosoftDocs/azure-devops-docs/issues/5890#issuecomment-539632059
     private const int MAX_SYSTEM_TITLE_LENGTH = 128;
     private const string TITLE_FIELD = "System.Title";
+    private static List<string> DEFAULT_REGRESSION_IGNORE_STATES = new() { "New", "Commited", "Active" };
```

Comment:
I was thinking about this from the opposite perspective: we only care about regression reports if the WI state is 'Closed', any other state we can ignore. I'm not asking you to change it since both ways achieve the same result, just floating the thought!

Reply:
yeah they are equivalent to me. Even if we invert the condition, we will still need to care about more than one state.
The example below we will need to care about Resolved, Closed, and Removed
![image](https://github.com/microsoft/onefuzz/assets/244326/ed6262f5-6346-48e7-8189-a6585f5778d3).

In our ado project we don't even have the closed state 
![image](https://github.com/microsoft/onefuzz/assets/244326/c6faaef6-9e7b-411f-a0c7-b7fed5898cdd)

Reply:
Ah that's a good point, in my head there was only one "done" state but it looks like that's not true